### ISSUE TITLE
notification.lua:117:in main chunk error

### ISSUE THREAD

**Issue by bezyking** [WillPower3309/awesome-dotfiles#2]
`Oops, an error happened! /home/x/.config/awesome/components/notification.lua:-117:attempt to call a nil value(field 'connect_signal')`
-----------------------------------------------------------
`Oops ,there were errors during startup! 
error while running function 
stack traceback:
      /home/x/.config/awesome/components/notification.lua:-117:in main chunk
      [C]:in function 'require' 
      /home/x/.config/awesome/rc.lua:-31:in main chunk
      error:  /home/x/.config/awesome/components/notification.lua:-117:attempt to call a nil value(field 'connect_signal')``

**Comment by WillPower3309** [comment]
Are you running the git version of awesome?

**Comment by vietduc01100001** [comment]
> Are you running the git version of awesome?

I'm having the same issue. I built awesome using the released `awesome-4.3.tar.bz2` file from their GitHub repo.
My `awesome -v` output:
```
awesome v4.3 (Too long)
 • Compiled against Lua 5.3.5 (running with Lua 5.3)
 • D-Bus support: ✘
 • execinfo support: ✔
 • xcb-randr version: 1.6
 • LGI version: 0.9.2
```

**Comment by WillPower3309** [comment]
I believe awesome has updated their notification widget making my notification.lua's formatting deprecated. I will look at it today and hopefully push an update tonight :)
